src/debug.c

```diff
@@ -1682,13 +1682,20 @@ void logStackTrace(void *eip, int uplevel) {
 void logServerInfo(void) {
     sds infostring, clients;
     serverLogRaw(LL_WARNING|LL_RAW, "\n------ INFO OUTPUT ------\n");
-    infostring = genRedisInfoString("all");
+    int out_all = 0;
+    int out_everything = 0;
+    robj *argv[1];
+    argv[0] = createStringObject("all", strlen("all"));
+    dict *section_dict = genInfoSectionDict(argv, 1, &out_all, &out_everything);
+    infostring = genRedisInfoString(section_dict, out_all, out_everything);
```

Comment:
I add a dict instead of using a NULL in the genRedisInfoString function according to your comments